src/patternfly/components/CodeEditor/examples/CodeEditor.md

```diff
@@ -113,6 +115,99 @@ cssPrefix: pf-v6-c-code-editor
 {{/code-editor}}
 ```
 
+
+### With optional code editor container 
+This is an extra container used in React to prevent event propagation if upload is enabled or there is a provided empty state.
+
+```hbs 
+{{#> code-editor}}
+  {{#> code-editor-container}}
+  {{#> code-editor-header}}
+    {{#> code-editor-header-content}}
+      {{#> code-editor-controls}}
+        {{> code-editor-controls--buttons}}
+      {{/code-editor-controls}}
+    {{/code-editor-header-content}}
+    {{#> code-editor-tab}}
+      {{#> code-editor-tab-icon}}
+        <i class="fas fa-code"></i>
+      {{/code-editor-tab-icon}}
+      {{#> code-editor-tab-text}}
+        HTML
+      {{/code-editor-tab-text}}
+    {{/code-editor-tab}}
+  {{/code-editor-header}}
+  {{#> code-editor-main}}
+    {{#> code-editor-upload}}
+      {{#> empty-state}}
+        {{#> empty-state-header}}
+          {{> empty-state-icon empty-state-icon--type="code"}}
+          {{#> empty-state-title}}
+            {{#> empty-state-title-text}}
+              Start editing
+            {{/empty-state-title-text}}
+          {{/empty-state-title}}
+        {{/empty-state-header}}
+
+        {{#> empty-state-body}}
+          Drag a file here or browse to upload.
+        {{/empty-state-body}}
+
+        {{#> empty-state-footer}}
+          {{#> empty-state-actions}}
+            {{#> button button--IsPrimary=true}}
+              Browse
+            {{/button}}
+          {{/empty-state-actions}}
+          {{#> empty-state-actions}}
+            {{#> button button--IsLink=true}}
+              Start from scratch
+            {{/button}}
+          {{/empty-state-actions}}
+        {{/empty-state-footer}}
+      {{/empty-state}}
+    {{/code-editor-upload}}
+  {{/code-editor-main}}
+  {{/code-editor-container}}
+{{/code-editor}}
+```
+
+
+### With full height modifier
+
+```hbs
+{{#> code-editor code-editor--modifier="pf-m-full-height"}}
+  {{#> code-editor-header}}
+    {{#> code-editor-header-content}}
+      {{#> code-editor-controls}}
+        {{> code-editor-controls--buttons}}
+      {{/code-editor-controls}}
+      {{> code-editor-header-main code-editor-header-main--text="Header main content"}}
+      {{#> code-editor-keyboard-shortcuts}}
+        {{#> button button--IsLink=true button--icon="question-circle"}}
+          View shortcuts
+        {{/button}}
+      {{/code-editor-keyboard-shortcuts}}
+    {{/code-editor-header-content}}
+    {{#> code-editor-tab}}
+      {{#> code-editor-tab-icon}}
+        <i class="fas fa-code"></i>
+      {{/code-editor-tab-icon}}
+      {{#> code-editor-tab-text}}
+        HTML
+      {{/code-editor-tab-text}}
+    {{/code-editor-tab}}
+  {{/code-editor-header}}
+  {{#> code-editor-main}}
+    {{#> code-editor-code}}
+      {{#> code-editor-code-pre}}
+        code goes here
+      {{/code-editor-code-pre}}
+    {{/code-editor-code}}
+  {{/code-editor-main}}
+{{/code-editor}}
+```
+
 ## Documentation
 
 ### Usage
```

Comment:
Could you add the `pf-m-full-height` modifier to the documentation section please?

Reply:
@rebeccaalpert can you add the `__container` element to the docs, too?

Reply:
You got it! All set @srambach and @mcoker!